package.xml

```diff
@@ -141,10 +140,15 @@
     
     
    	<target name="archiveSkipTests">
-		 <property name="RUN_NO_TESTS" value="Yes"/>
-		<antcall target="archives"/>
-	</target>
+		   <property name="RUN_NO_TESTS" value="Yes"/>
+		   <antcall target="archives"/>
+      </target>
 
+   	<target name="archiveSkipTestsAndBuildSubProjects">
```

Comment:
I think the name of this target is too easy to read ambiguously:  it could be parsed as either "do an archive, skip the tests, and Build the Subprojects", or "do an archive but skip the Tests and the building of subprojects.   In `build.xml`, the analogous target name is (should be corrected to be) `jarSkipTestsAndSubprojectBuilds`; an analogous (and better) name for this target here would be `archiveSkipTestsAndSubprojectBuilds`, which is much harder to parse incorrectly.

Reply:
pushed fix.